server/text_generation_server/layers/fp8.py

```diff
@@ -79,11 +82,37 @@ def normalize_e4m3fn_to_e4m3fnuz(
     return weight, weight_scale, input_scale
 
 
+def per_tensor_dequantize(
+    tensor: torch.Tensor, inv_scale: Union[float, torch.Tensor]
+) -> torch.Tensor:
+    fake_qweight = tensor.to(torch.float16)
```

Comment:
I think this should be the model dtype? (The quantized numbers could represent values that are only in `bfloat16`'s range.)

Reply:
Yes made the changes